src/components/views/rooms/EventTile.js

```diff
@@ -318,12 +325,12 @@ module.exports = React.createClass({
                 avatar = (
                     <div className="mx_EventTile_avatar">
                         <MemberAvatar member={this.props.mxEvent.sender} width={24} height={24}
-                         onClick={ this.onMemberAvatarClicked.bind(this, this.props.mxEvent.sender) } />
+                         onClick={ this.onMemberAvatarClick } />
                     </div>
                 );
             }
             if (EventTileType.needsSenderProfile()) {
-                sender = <SenderProfile mxEvent={this.props.mxEvent} aux={aux} />;
+                sender = <SenderProfile onClick={ this.onSenderProfileClick } mxEvent={this.props.mxEvent} aux={aux} />;
```

Comment:
is there an associated vector-web change to add an onClick property to SenderProfile?